core/modules/system/system.module

```diff
@@ -4056,10 +4056,9 @@ function path_autocomplete($string = '') {
       ->execute();
     foreach ($results as $result) {
       $path = backdrop_get_path_alias('node/' . $result->nid);
-      $matches[$path] = t('@title (Content: %type)', array(
-        '@title' => $result->title,
-        '%type' => node_type_get_name($result->type),
-      ));
+      $match = '<span class="autocomplete-suggestion">' . $result->title . '</span>';
+      $match_description = t('Content: %type', array('%type' => node_type_get_name($result->type)));
+      $matches[$path] = $match . " " . '<span class="autocomplete-description">(' . $match_description . ')</span>';
```

Comment:
```suggestion
      $matches[$path] = $match . ' <span class="autocomplete-description">(' . $match_description . ')</span>';
```

Reply:
Thanks 👍 ...although this is not specifically mentioned in https://api.backdropcms.org/php-standards#string-concatenation, I've fixed it with [`a30ebb0` (#3189)](https://github.com/backdrop/backdrop/pull/3189/commits/a30ebb0d06f134132143d111fc2e1b3ae2a252ec)